script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -156,17 +157,17 @@ It troubles me if you go out of your way to wait far before our time like that."
 [sha:8595a8e0817de0661919124e29bb9c31c7821d71]{
 -- Page 302, Offset 20420.
 -- しっかし、なんだって３時間も待ってたんだ。間違えて来たにしろ、それだけあれば部屋に戻っていても良かったじゃないか」
-I mean, why did you even wait for three hours? Even if you made a mistake, you could have just returned to your room."
+I mean, why did you even wait for three hours? Even if you made a mistake, you could have just gone back to your room."
```

Comment:
mark this as an italics candidate "why /did/ you wait for three hours? Even if you" to remove the "even". no change for now, just a comment